src/main/java/by/custom/utilcalculator/domain/UserProgress.java

```diff
@@ -12,17 +14,23 @@ public class UserProgress implements Serializable {
     private VolumeOfEngine volumeOfEngine = null;
     private final String chatID;
     private Step currentQuestion;
+    private Enum[] stepsData;
```

Comment:
If I right understand the intension `chosenAnswers` or just `answers` is more suitable.

Reply:
I prefer to use Commands here. Anyway, my naming is wrong